### ISSUE TITLE
Dark mode lost when moving to the right of standard window size

### ISSUE THREAD

**Issue by martin-networker** [nautobot/nautobot#3509]
### Environment
* Nautobot version (Docker tag too if applicable): 1.5.9
* Python version: 3.10.9
* Database platform, version:
* Middleware(s):

### Steps to Reproduce
1. Open IP Addresses object in GUI
2. scroll down to Related IP Addresses
3. If related IP addresses has more parameters there will be scroll bar allowing to move to the right.
4. When scrolling to the right there will be no dark mode.
 

![image](https://user-images.githubusercontent.com/84594831/228489756-79a1b08b-9661-43ce-bfe1-4f83facdf8f1.png)

**Comment by UnicodeTreason** [comment]
We're also seeing this when scrolling down.

Nautobot Version: v1.5.20
Python version: 3.9
Platform: Container (nautobot:1.5.20-py3.9)

**Comment by gsnider2195** [comment]
Potentially related? We're seeing issues with the footer and the Parent Prefixes panel on the IPAddress detail view in the current `next` commit

![image](https://github.com/nautobot/nautobot/assets/75227981/8c170ac2-e005-4c80-9968-3621f8b582ae)

![image](https://github.com/nautobot/nautobot/assets/75227981/47a4cab4-6385-4105-aaa4-8fe44d671310)

**Comment by norbert-mieczkowski-codilime** [comment]
Yeah it's likely an overflowing issue when desendants are larger than their ancestors. Since the background color is set only on the container elements and children are larger than them, some of their parts are drawn over an empty space which has no color and that's why it's not visible in the light mode. But with `table-responsive` class this should no longer happen because it restricts tables from overflowing their parents.

**Comment by itdependsnetworks** [comment]
per @norbert-mieczkowski-codilime response, I believe this is fixed, feel free to open back up if I got that incorrect.
